db/scripts/seed_database.js

```diff
@@ -33,6 +33,12 @@ async function seedDatabase() {
     VALUES ('Alice', 'bob@example.com', 'Bob', CURRENT_TIMESTAMP + interval '1 hour', '123', 'whereby', $1, 'password', 'scheduled')`,
     [wardId]
   );
+  await db.result(
```

Comment:
Do we need another visit in the seeds or can we re-use the existing one ?

Reply:
Added a new one so that the error test in the Cypress test didn't rely on the patient name change of the happy path test.